.github/workflows/autobuild.yml

```diff
@@ -157,35 +148,32 @@ jobs:
              cmd1_prebuild:         powershell .\autobuild\windows\autobuild_windowsinstaller_1_prepare.ps1
              cmd2_build:            powershell .\autobuild\windows\autobuild_windowsinstaller_2_build.ps1
              cmd3_postbuild:        powershell .\autobuild\windows\autobuild_windowsinstaller_3_copy_files.ps1
-             uses_codeql:           true
+             run_codeql:            true
 
            - config_name:           Windows JACK (artifact)
              target_os:             windows
              building_on_os:        windows-2019
              cmd1_prebuild:         powershell .\autobuild\windows\autobuild_windowsinstaller_1_prepare.ps1 -BuildOption jackonwindows
              cmd2_build:            powershell .\autobuild\windows\autobuild_windowsinstaller_2_build.ps1 -BuildOption jackonwindows
              cmd3_postbuild:        powershell .\autobuild\windows\autobuild_windowsinstaller_3_copy_files.ps1 -BuildOption jackonwindows
-             uses_codeql:           false
+             run_codeql:            false
 
     runs-on:                        ${{ matrix.config.building_on_os }}
     steps:
-
       - name:                       Select Xcode version for Mac
-        if:                         ${{ matrix.config.target_os == 'macos' || matrix.config.target_os == 'ios' }}
+        if:                         matrix.config.target_os == 'macos' || matrix.config.target_os == 'ios'
         uses:                       maxim-lobanov/setup-xcode@v1
         with:
-          xcode-version: ${{ matrix.config.xcode_version }}
+          xcode-version:            ${{ matrix.config.xcode_version }}
 
-      # Checkout code
       - name:                       Checkout code
         uses:                       actions/checkout@v2
         with:
           submodules:               true
           fetch-depth:              ${{ matrix.config.checkout_fetch_depth || '1' }}
 
```

Comment:
Set up build environment. This downloads dependencies if needed or uses those cached from previous runs.

Reply:
This sounds like some kind of headline to me, but technically there is no such thing. All of that is done as part of the `release_assets` job. It doesn't feel right to add such a "block comment" in a place where there is no block.

What are you mainly after at? Maybe adding the "Download dependencies" part to the rather inaccurate "Prepare" step later would be beneficial? I'll add a suggestion below.

I'm all for clarifying the structure of this file. Github's workflow syntax already provides us with a way to technically annotate each step (`name:`). I think its use is even mandatory. Therefore, I think we should do all the useful optimizations there and don't add another level of structure via comments.
Improving `name:` has the benefits of also improving the CI logs in the same go. A `grep '[ -] name:' .github/workflows/autobuild.yml | grep -v artifact` can also be used to get a sense of the structure of this workflow.

Reply:
> Improving name: has the benefits of also improving the CI logs

Good point. And yes 'Prepare' sounds impercise.